tests/network/probes.go

```diff
@@ -67,7 +67,9 @@ var _ = SIGDescribe("[ref_id:1182]Probes", func() {
 		httpProbe := createHTTPProbe(period, initialSeconds, port)
 		guestAgentPingProbe := createGuestAgentPingProbe(period, initialSeconds)
 
-		DescribeTable("should succeed", func(readinessProbe *v1.Probe, ipFamily corev1.IPFamily, isExecProbe bool, disableEnableCycle bool) {
+		DescribeTable("should succeed", func(readinessProbe *v1.Probe, ipFamily corev1.IPFamily, disableEnableCycle bool) {
+			isExecProbe := readinessProbe.Exec != nil
```

Comment:
Great stuff!
nit: Maybe a function definition would look even better and allow for code reuse?
`isExecProbe := isExecProbe(readinessProbe.Exec)`?